shared/actions/chat/inbox.js

```diff
@@ -104,13 +104,16 @@ function* onInboxStale(): SagaGenerator<any, any> {
       }
     )
 
-    const incoming = yield loadInboxChanMap.race({removeNs: true})
-    if (!incoming.chatInboxUnverified || !incoming.chatInboxUnverified.response) {
+    const incoming = yield loadInboxChanMap.race()
+    if (
+      !incoming['chat.1.chatUi.chatInboxUnverified'] ||
```

Comment:
🐑  we could stash these in a variable so it could be more simliar to the old stuff
```
const chatInboxUnverified = 'chat.1.chatUi.chatInboxUnverified'
incoming[chatInboxUnverified].response() // etc
```